tests/e2e/e2e_suite_test.go

```diff
@@ -295,6 +298,28 @@ func (s *IntegrationTestSuite) executeQueries() {
 	s.T().Logf("executed all queries successfully")
 }
 
+// executeTransactions executes some sample transactions to check they are still working after the upgrade.
+func (s *IntegrationTestSuite) executeTransactions() {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	chainID := utils.TestnetChainID + "-1"
+
+	// send some tokens between accounts to check transactions are still working
+	exec, err := s.upgradeManager.CreateModuleTxExec(upgrade.E2eTxArgs{
+		ModuleName: "bank",
+		SubCommand: "send",
+		Args:       []string{"mykey", "evmos1jcltmuhplrdcwp7stlr4hlhlhgd4htqh3a79sq", "10000000000aevmos"},
+		ChainID:    chainID,
+		From:       "mykey",
+	})
+	s.Require().NoError(err, "failed to create bank send tx command")
+
+	_, errBuf, err := s.upgradeManager.RunExec(ctx, exec)
```

Comment:
Nit: check that the transfer was successful (e.g. check destination/origin address balance changed)